client/webpack.config.js

```diff
@@ -392,6 +393,37 @@ const webpackConfig = {
 		new ExtensiveLodashReplacementPlugin(),
 	].filter( Boolean ),
 	externals: [ 'keytar' ],
+	...( shouldUsePersistentCache
+		? {
+				cache: {
+					// eslint-disable-next-line inclusive-language/use-inclusive-words
+					// More info in https://github.com/webpack/changelog-v5/blob/master/guides/persistent-caching.md
+					type: 'filesystem',
+					version: [
```

Comment:
The version intentionally doesn't contain a reference to the commit hash because we want the cache to transcend beyond it, right?

Reply:
Exactly right.